src/drawer.multicanvas.js

```diff
@@ -106,8 +106,9 @@ export default class MultiCanvas extends Drawer {
      */
     updateSize() {
         const totalWidth = Math.round(this.width / this.params.pixelRatio);
+        // No need more canvas if it's only for 2 peaks (there the same for overlaps !)
         const requiredCanvases = Math.ceil(
-            totalWidth / this.maxCanvasElementWidth
+            totalWidth / (this.maxCanvasElementWidth + 2)
```

Comment:
By "2 peaks", do you mean "2 pixels"?

Reply:
Hi @chrisparton1991 , I really say 2 peaks.
TotalWidth it's a pixelRatio, not a real pixel. Than number 2 it's for peaks and for caluted the number of canvas need.
You can see #1399 picture result have more then 2 pixels for 2 peaks.

Reply:
So if I understood correctly, the extra pixel (or two subpixels) is a result of a rounding error somewhere in the zoom code?

Reply:
Not really a rounding Error, but this différence clearliy appear on Zoom process like my screen shot I have posted. Because zooming add a lot of canvas and error it's on the join position af the cnavas. Perha^ps @chrisparton1991 can explian this better than me, but in Multicanvas each canvas begin on same position of the end of previous canvas, than first peak it's the same than the last, and last it's the same of the first of the next. Thans if you just have same number of peaks

Reply:
Gotcha! (I think)
If it works, then it works 👍